### DISCUSSION TITLE
Discord signal to TradingView possible

### DISCUSSION THREAD

**Discussion by jryals579** [FabCx/TradingView-Webhook-Bot#122] [Q&A] [answered]
I have seen a trading view indicator that links to a discord server, on the discord server i was a member and they had alerts/ signals that I could like so it would notify me on discord but it also would show up on trading view on the chart . The idea is a signaling group post their entry , stop loss etc and that would somehow make it to TradingView. Is this possible with your code ?

**Comment by fabston** [comment, marked as answer]
The code as it is right now does not support that but with a bit of extra code it is possible, yes.

**Reply by jryals579** [reply]
Thank you for contributing your excellent code. I respect coders on ALL levels ... LOL I am in IT but different aspect
